### ISSUE TITLE
Multi-page forms focus -- keyboard nav issue

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#7279]
See the [ticket raised in the support channel](https://gcdigital.slack.com/archives/C03T4FAE9FV/p1779999578418029) for more info.

On a multi-page form, when using a keyboard to navigate the last form element can some times be blocked by the navigation bar. (see video)

https://github.com/user-attachments/assets/e5970768-4104-4f5e-a214-d8c709833249
